packages/account-v2/src/containers/POAForm/AddressDetailsForm.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { Fragment } from 'react';
+import { Form, Formik } from 'formik';
+import { useHistory } from 'react-router-dom';
+import { useDocumentUpload, useSettings } from '@deriv/api';
+import { useBreakpoint } from '@deriv/quill-design';
+import { StandaloneXmarkBoldIcon } from '@deriv/quill-icons';
+import { Button } from '@deriv-com/ui/dist/components/Button';
+import { Text } from '@deriv-com/ui/dist/components/Text';
+import { InlineMessage } from '../../components/base/InlineMessage';
+import { AddressFields } from '../../modules/AddressFields';
+import DocumentSubmission from './DocumentSubmission';
+import { POAError } from './Status';
+
+type TAddressDetails = {
+    addressCity: string;
+    addressLine1: string;
+    addressLine2: string;
+    addressPostcode: string;
+    addressState: string;
+    document?: File;
+};
+
+type TAddressDetailsForm = {
+    resubmitting?: boolean;
+};
+
+export const AddressDetailsForm = ({ resubmitting }: TAddressDetailsForm) => {
+    const { data: settings, error: fetchError, mutation, update: updateSettings } = useSettings();
```

Comment:
**NIT: ** Can we alias mutation to something more understandable? Or please add a line of comment on what the mutation represents

Reply:
Changed the way we are destructuring.